main.cpp

```diff
@@ -130,25 +237,45 @@ int main()
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
-    //1)
+    //1)
+    guessTheNumber(7, 3);
+    ignoreUnused(guessTheNumber);
     
-    //2)
+    //2)
+    doSomeMath(1, 300, 3.14f);
+    ignoreUnused(doSomeMath);
     
     //3)
+    checkTheStatus(1);
+    ignoreUnused(checkTheStatus);
+
+    //4)
+    questionExistence(28, 45);
+    ignoreUnused(questionExistence);

```

Comment:
your functions are not `unused`.  If they return anything, store the returned value in a variable, and pass that variable to `ignoreUnused`
- [ ] revise